BE/src/main/java/louie/dong/airbnb/book/dto/BookDetailResponse.java

```diff
@@ -17,7 +20,22 @@ public class BookDetailResponse {
 	private String accommodationCountry;
 	private String accommodationName;
 	private int guestCount;
-	private String roomCount;
+	private RoomType roomType;
 	private String hostName;
 	private int finalPrice;
+
+	public BookDetailResponse(Book book) {
+		this.bookId = book.getId();
+		this.accommodationImageUrls = book.getAccommodation().getAccommodationImages().stream()
+			.map(AccommodationImage::getImageUrl).collect(Collectors.toList());
+		this.checkIn = book.getCheckIn();
+		this.checkOut = book.getCheckOut();
+		this.accommodationCountry = book.getAccommodation().getCountry();
+		this.accommodationName = book.getAccommodation().getName();
+		this.guestCount = book.getGuestCount();
+		this.roomType = book.getAccommodation().getRoomInformation().getRoomType();
+		this.hostName = book.getAccommodation().getHostName();
```

Comment:
메소드 체이닝으로 내부 데이터를 조회하는 경우 book 클래스에 메소드를 만들어줘도 좋을것 같습니다.
```suggestion
		this.roomType = book.getRoomType();
		this.hostName = book.getHostName();
```

Reply:
조회 메소드를 내부에서 선언하는 경우 null point exception 방어하기 좋습니다.